endpoint_info.go

```diff
@@ -211,9 +211,6 @@ func (ep *endpoint) AddStaticRoute(destination *net.IPNet, routeType int, nextHo
 }
 
 func (ep *endpoint) addInterfaceRoute(route *types.StaticRoute) error {
-	ep.Lock()
-	defer ep.Unlock()
-
```

Comment:
What about getting rid of this function completely ? It is used only in `AddStaticRoute()` and can be done in one line of code.